### PR TITLE
chore: updated land fixed links on Clarinet page

### PR DESCRIPTION
**Pull request by pgray-hiro**
## Description

This PR fixes a few links on the Clarinet page.

## Checklist

- [x] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] New links to files and images were verified
- [x] For fixes/refactors: all existing references were identified and replaced
- [x] [Style guide](https://developers.google.com/style) was reviewed and applied
- [x] Clear code samples were provided
- [x] People were tagged for review

### CODE REVIEW

File: src/pages/write-smart-contracts/clarinet.md [hunk 1]
```diff
@@ -184,8 +184,8 @@ Clarinet.test({
 });
 ```
 
-For more information on assertions, review [asserts][] in the Deno standard library. For more information on the available
-Clarity calls in Deno, review the [Deno Clarity library][].
+For more information on assertions, review [asserts][] in the Deno standard library. For more information on the
```
Comment (agraebe) [inline src/pages/write-smart-contracts/clarinet.md:187]: looks like the URLs are missing, right?

Comment (pgray-hiro) [reply src/pages/write-smart-contracts/clarinet.md:187]: These are [reference-style Markdown links](https://daringfireball.net/projects/markdown/syntax#link). I think they make the raw Markdown easier to read. You can see where they are defined at the bottom of the file.

Comment (agraebe) [reply src/pages/write-smart-contracts/clarinet.md:187]: wow, **TIL**

**Review by agraebe — APPROVED** [review]
LGTM

**Pull request merged**